Tests/test_imagesequence.py

```diff
@@ -23,14 +23,11 @@ def test_sanity(self):
         self.assertEqual(index, 1)
 
     def _test_multipage_tiff(self, dbg=False):
-        # debug had side effect of calling fp.tell.
-        Image.DEBUG=dbg
         im = Image.open('Tests/images/multipage.tiff')
```

Comment:
Note: "debug had side effect of calling fp.tell." This was from PR https://github.com/python-pillow/Pillow/pull/885 fixing https://github.com/python-pillow/Pillow/issues/862. This may or may not still be needed once the other things are passing.